tests/client/test_machine.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import tinychain as tc
+import unittest
+from sklearn.model_selection import train_test_split
+from testutils import ClientTest
+
+Dense = tc.tensor.Dense
+ENDPOINT = "/transact/hypothetical"
+LEARNING_RATE = tc.F32(0.01)
+MAX_ITERATIONS = 10
+NUM_EXAMPLES = 20
+
+
+class Logistic_Regression():
```

Comment:
I still don't see the advantage of making this a class. You can just delete this line and then it will be a function.